### DISCUSSION TITLE
全局更改组件的empty有具体的使用方法吗？能否补充修改的文档说明？

### DISCUSSION THREAD

**Discussion by maddenBoy** [tusen-ai/naive-ui#6119] [Q&A] [answered]
全局更改组件的empty有具体的使用方法吗？
看源码发现给n-config-provider绑定:component-options="{ Empty: { description: '测试' } }"属性可以修改，能否补充修改的文档说明？不然使用者很难知道怎么全局修改组件的empty
![image](https://github.com/tusen-ai/naive-ui/assets/71129660/74d03f96-f16f-4358-9525-8ba63084f3ea)

**Comment by pengzhen-developer** [comment]
参考：
https://github.com/tusen-ai/naive-ui/issues/3232

**Comment by jahnli** [comment, marked as answer]
https://github.com/tusen-ai/naive-ui/pull/7342
